lib/rihanna/job.ex

```diff
@@ -356,7 +356,16 @@ defmodule Rihanna.Job do
   def after_error(job_module, reason, arg) do
     if :erlang.function_exported(job_module, :after_error, 2) do
       # If they implemented the behaviour, there will only ever be one arg
-      job_module.after_error(reason, arg)
+      try do
+        job_module.after_error(reason, arg)
+      rescue
+        exception ->
+          Logger.log(:debug, fn ->
```

Comment:
This should probably be a warning.

Curious about the choice of `Logger.log` over `Logger.debug`